### DISCUSSION TITLE
Multiuser

### DISCUSSION THREAD

**Discussion by ferdinandSoethe** [super-productivity/super-productivity#3707] [Q&A]
We love SP, thanks for all the effort that went into it.
Now we would like to use it to manage tasks in our familiy business and I would like to ask, if there is a way to assign users to a task and add a view that shows just the tasks, schedules etc that are assigned to a certain user.

Thanks for your help.

**Comment by johannesjo** [comment]
Hey there! What is your setup? Do you work from the same computer? Is using third party online services an option? What are your requirements? 

Super Productivity itself is more of a single user application, but you can integrate other tools that allow for cooperation, such as open project, redmine and others.

**Reply by ferdinandSoethe** [reply]
I'd like to use SP with a file with me Nextcould-Server. We work from different PCs and I'd like to use the Android-App as well. Didn't get it to replicate with Nextcould yet.
My requs are to manage todos for 2-3 people an have additional view for each person.

Kind regards
Ferdinand

**Reply by johannesjo** [reply]
One idea would be to use nextcloud as a WebDAV server for this. No idea how well this would work.

**Reply by ferdinandSoethe** [reply]
That seems to work from Windows. Android needs more experiments ...
But is the synching mechanism able to lock and update with more than one client?
AND biggest question: how can I add user assignement for a task and filter views for that user?

Kind regards
Ferdinand

**Reply by johannesjo** [reply]
Syncing is only meant to be used by a single user only. It is syncing between different devices not users.

> AND biggest question: how can I add user assignement for a task and filter views for that user?

You can't. Super Productivity is meant to integrate with all kinds of third party tools, but apart from that it is a tool to organize your personal work, not teams. There are other tools for that.

**Reply by mfvanrijn** [reply]
I just found this app and was immediately wondering how I can see what my team is working on! I understand that this app integrates with other tools, but it would be amazing if this kind of simplicity was part of my team's way of working without going into other apps.

**Reply by dominik3bb** [reply]
Didn‘t expect this much demand for that usecase. Maybe a new plugin would be able to „scratch that itch“ by adding this experimental feature?

**Comment by mredodos** [comment]
pls add multi user support

**Comment by Mephisztoe** [comment]
After Wunderlist was closed my family migrated over to Microsoft ToDo to manage regular household tasks. We have different accounts but share one tasklist that features regular, daily, weekly, monthly appearing tasks. I have an iPhone, my wife has an Android Smartphone. However, the sync between the different platforms is horrible... which is why my wife started to transfer the digitally managed tasks onto a physical blackboard--- daily (so that I know what's on _my_ list).

I was wondering if SP is able to handle this use case better... and found this Multi User feature request.
It seems as if exactly this one point is missing, so I second this request. ;-)

**Reply by darius-r** [reply]
Unless I'm missing something, there appears to be an immediately usable solution for you with minimal adaptation. 

Unlike all of those cloud based SaaS (Software as a Service) products you listed, Super Productivity is explicity not cloud based. It's local first and centered around a file which you can opt to sync across devices with Dropbox, Nextcloud, etc. 

If neither of you are adverse to sharing similar views within the app, you could simply work in the same file across devices. Customized adaptation could be achieved using tags (your names perhaps), Custom board views using project and/or tag filters, and the Automation feature (which would allow for auto tagging tasks based on the Project their created in). 

Tangentially, there's an experimental 'multi profile' feature. I can't help but think this would be getting considered in tandem with multi user, as it could potentially be one path to achieve it.
